app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
Use `enumerate()` if you want indexes of list elements. Same in two other places below.

```suggestion
    for index, arg in enumerate(argv[:-1]):
```